webpack.config.babel.js

```diff
@@ -0,0 +1,48 @@
+import webpack from 'webpack';
+import path from 'path';
+
+const BUILD_PATH = path.resolve(__dirname, 'public');
+const APP_DIR = `${path.resolve(__dirname)}/client`;
+
+export default {
+  entry: `${APP_DIR}/index.js`,
+  output: {
+    path: BUILD_PATH,
+    filename: 'bundle.js'
+  },
+  module: {
```

Comment:
webpack 3 uses rules I believe, and we can combine `js` and `jsx` using `test: /\.jsx?$` regex